modules/machineset-vsphere-required-permissions.adoc

```diff
@@ -68,7 +68,7 @@ If you cannot use an account with global administrative privileges, you must cre
 `VirtualMachine.Provisioning.Clone`
 
 |vSphere vCenter data center
-|If the installation program creates the virtual machine folder
```

Comment:
This was modified to language that doesn't make much sense in vSphere. This should be switched back